_1327/templates/base.html

```diff
@@ -48,6 +48,13 @@
 						{% endif %}
 					</li>
 					{% endfor %}
+					{% if CAN_CREATE_INFORMATIONPAGE %}
+						<li class="pull-right">
+							<a href="{% url 'information_pages:create' %}">
+								<span class="glyphicon glyphicon-plus" aria-hidden="true"></span>
+							</a>
+						</li>
+					{% endif %}
```

Comment:
We can't create such a link for each type of document, see above.
What about one link if the user can create *any* type of document?